src/common/transformations/src/transformations/op_conversions/convert_mvn1_to_mvn6.cpp

```diff
@@ -31,13 +31,18 @@ ov::pass::ConvertMVN1ToMVN6::ConvertMVN1ToMVN6() {
         if (input_rank.get_length() <= start_axis) {
             return false;
         }
+
+        const auto eps = mvn_node->get_eps();
+        if (eps > std::numeric_limits<float>::max() || eps < std::numeric_limits<float>::min())
+            return false;
```

Comment:
The `ConvertMVN1ToMVN6` pass is registered only for GNA plugin, so it's worth to ensure that `return false` in such case doesn't break backward compatibility.
Another solution that can be considered is to use the `numeric_limits<float>` value as new epsilon, especially when the input/output element_type is f32.

Reply:
@evkotov @kbruniec Would you advise?

Reply:
guys, don't you know why `double` for `eps` is changed to `float`? Currently, we seem to need `numeric_limits<float>` for such cases. If we have side effects for it, we need to consider new spec with `double`

Reply:
eps > std::numeric_limits<float>::max()  : float_eps = std::numeric_limits<float>::max();
eps < std::numeric_limits<float>::min() : float_eps =  std::numeric_limits<float>::min();
something like that I think?
but please check that it's correct to compare 2 double (float) numbers like that (eps > std::numeric_limits<float>::max())

Reply:
`std::numeric_limits<>::epsilon()` is too big for min limit and too small for max. I used `std::nextafter()` for limits comparison.

Reply:
@rkazants I presume your question concerns MVN operator definition itself. This conversion is now used only by GNA plugin, where it's not a case.